src/rulesets/oas/index.json

```
@@ -569,7 +569,7 @@
       "then": {
         "function": "pattern",
         "functionOptions": {
-          "notMatch": "/$"
+          "notMatch": ".+/$"
```

```suggestion
          "notMatch": "./$"
```
IMHO we don't need `+` quantifier here.
We're here only to verify whether there is at least a single character placed in front of the slash.
We don't care about the amount.